src/components/BaseWizard/Layouts/VerticalLayout/VerticalLayout.vue

```diff
@@ -22,8 +22,8 @@
       </template>
     </NavMobile>
 
-    <header v-if="title || $slots.header">
-      <slot name="title">
+    <header v-if="$slots.title || title || $slots.header">
+      <slot v-if="$slots.title || title" name="title">
```

Comment:
I think this might be what we want to do. 
```suggestion
    <header>
      <slot v-if="$slots.title || title" name="title">
```